### ISSUE TITLE
How to use f1_score from sklearn?

### ISSUE THREAD

**Issue by saja1994** [RandolphVI/Multi-Label-Text-Classification#5]
Thank you for sharing this helpful code. please, I want to ask you if you can help me to implement f1_score from sklearn into your implementation to measure the f score because I got low f1 values using your code and want to try different f1 average ways like macro and micro, and compare them.

Thanks a lot.

**Comment by RandolphVI** [comment]
Hi there,

What you mentioned, implement f1_score() from sklearn, it’s important since users want to calculate the micro-f1 or macro (actually what I implement is called sampled-f1, but is not used basically).

And I have implemented the code already, and I will update the code in one or two days (a little bit busy recently, ddl).

Hope this helps!

Randolph

在 2019年1月29日，上午12:21，saja1994 <notifications@github.com<mailto:notifications@github.com>> 写道：


Thank you for sharing this helpful code. please, I want to ask you if you can help me to implement f1_score from sklearn into your implementation to measure the f score because I got low f1 values using your code and want to try different f1 average ways like macro and micro, and compare them.

Thanks a lot.

**Comment by saja1994** [comment]
Thank you so much, I really appreciate your effort.

**Comment by RandolphVI** [comment]
Hi @saja1994 ,

I have updated the code already, now you can predict the evaluation metrics using sklearn.

**Comment by saja1994** [comment]
Thank you so much.
